### DISCUSSION TITLE
Add option to skip ordering of issues so ORDER BY in JQL can be used for more complex sorting

### DISCUSSION THREAD

**Discussion by 20manas** [ankitpokhrel/jira-cli#771] [Ideas]
For example, I want to `ORDER BY resolution DESC, rank` but I can't do it with `--order-by`.  If I pass `ORDER BY` to `--jql`, it doesn't work because the cli is always passing its own `ORDER BY`. So if there is a functionality to stop automatic passing of `ORDER BY`, it will allow the users to pass their own order in the jql string.

I can see a few ways to implement this:
- Add `--no-order` or `--skip-order`
- Add `--order-by 'somevalue'` which will skip sorting. `somevalue` can be `null`, `nil`, `empty` or something else
- If `ORDER BY` is present in the jql string, the cli should not pass its own `ORDER BY`

Thanks!

Edit: Actually, for my specific case, I was able to do it with `--order-by 'resolution DESC,rank' --reverse`. Although this functionality can still be useful for other queries/scenarios, so leaving this discussion open for now.

**Comment by HarrisonMc555** [comment]
Alternatively, we could also allow users to provide a finer-grained control of the ordering via the existing `--order-by` flag. I'm not sure of the best syntax, but something like this would be feasible:

```
jira issue list --order-by 'priority DESC, status ASC'
```

...in fact, this is already _kind of_ supported...in a very janky way.

I'm guessing that the argument to the `--order-by` argument is simply interpolated into a raw JQL query string. I'm guessing that a `DESC` or `ASC` flag is appended to the end (as determined by the presence or absence of the `--reverse` flag).

So, you can actually specify arbitrarily complex `ORDER BY` clauses by simply adding in the `ASC` or `DESC` clauses to your `--order-by` argument, as long as the final argument _doesn't_ have a `ASC` or `DESC` clause.

For example, this works:

```
jira issue list --order-by 'priority DESC,status ASC,duedate ASC,created'
```

(note that the final `created` column does _not_ have an `ASC` or `DESC` modifier)

This is a weird workaround. It would be nice if this were supported in a better way. I think the best way would probably be to either recognize if the final "word" was `ASC` or `DESC` (case-insensitively) and refrain from adding an additional `ASC`/`DESC` if so, or to deprecate the `--reverse` flag and require users to specify `ASC` or `DESC` themselves.
